src/main/java/org/webeid/security/validator/AuthTokenValidationConfiguration.java

```diff
@@ -53,7 +53,12 @@
     private boolean isSiteCertificateFingerprintValidationEnabled = false;
     private String siteCertificateSha256Fingerprint;
     // Don't allow Estonian Mobile-ID policy by default.
-    private Collection<ASN1ObjectIdentifier> disallowedSubjectCertificatePolicies = Sets.newHashSet(EST_MOBILE_ID_POLICY);
+    private Collection<ASN1ObjectIdentifier> disallowedSubjectCertificatePolicies = Sets.newHashSet(
+        ESTEID_SK_2015_OLD_MOBILE_ID_POLICY_V1,
+        ESTEID_SK_2015_OLD_MOBILE_ID_POLICY_V2,
+        ESTEID_SK_2015_OLD_MOBILE_ID_POLICY_V3,
+        ESTEID_SK_2015_NEW_MOBILE_ID_POLICY
```

Comment:
Can we remove the OLD/NEW and the version suffix is already good

Reply:
Thanks, good idea, will do!

Reply:
Done.